core/src/main/java/com/linecorp/armeria/client/retry/RetryingClient.java

```diff
@@ -340,6 +345,26 @@ private void doExecute0(ClientRequestContext ctx, HttpRequestDuplicator rootReqD
         }
     }
 
+    private void handleMaybeErrorResponse(RetryConfig<HttpResponse> config, ClientRequestContext ctx,
```

Comment:
Suggestion: I found it difficult to understand the meaning of this method.
It looks to handle a response without a response body content.
How about naming it `handleResponseWithoutContent()`?

Reply:
Renamed it. 😉